nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-content-viewer/src/main/java/org/apache/nifi/web/ContentViewerController.java

```diff
@@ -150,6 +150,7 @@ protected void doGet(final HttpServletRequest request, final HttpServletResponse
         // buffer the content to support reseting in case we need to detect the content type or char encoding
         try (final BufferedInputStream bis = new BufferedInputStream(downloadableContent.getContent());) {
             final String mimeType;
+            final String normalizedMimeType;
 
             // when standalone and we don't know the type is null as we were able to directly access the content bypassing the rest endpoint,
             // when clustered and we don't know the type set to octet stream since the content was retrieved from the node's rest endpoint
```

Comment:
@soenkeliebau - I overlooked this the first time I reviewed because the GitHub diff view had it collapsed. I believe we need to modify the check on the next line that will allow Tika content type detection when the type from the DownableContent is null or if the value starts with ignore case "application/octet-stream".

This is a simple change I can make after applying your commit so I don't think we need to go through the process of re-submitting the PR, I just wanted to make sure you agree and we're on the same page.